src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1376,17 +1368,10 @@ public void canPageLoans()
     loansFixture.checkOutByBarcode(itemsFixture.basedUponUprooted(),user);
     loansFixture.checkOutByBarcode(itemsFixture.basedUponInterestingTimes(),user);
 
-    CompletableFuture<Response> firstPageCompleted = new CompletableFuture<>();
     CompletableFuture<Response> secondPageCompleted = new CompletableFuture<>();
 
-    client.get(loansUrl() + "?limit=3",
-      ResponseHandler.json(firstPageCompleted));
-
-    client.get(loansUrl() + "?limit=3&offset=3",
-      ResponseHandler.json(secondPageCompleted));
-
-    Response firstPageResponse = firstPageCompleted.get(5, TimeUnit.SECONDS);
-    Response secondPageResponse = secondPageCompleted.get(5, TimeUnit.SECONDS);
+    Response firstPageResponse = loansFixture.getLoans(3);
+    Response secondPageResponse = loansFixture.getLoans(3, 3);
```

Comment:
Can we rename to something `getLoansWithOffset` or similar?

It is not so clear for me what `getLoans(3, 3)` does.

Reply:
I've tried a different approach, with strongly typed parameters instead. I'd be interested in your thoughts.

The implementation is very limited so far, it mostly is a different way of documenting the parameters and stops mistakes like mixing up the parameters. In the future it can be extended to include how to describe itself as a parameter and to put range limits on the values (e.g. they should never be negative)

Reply:
Thanks, looks good.